client/package.json

```diff
@@ -32,9 +32,9 @@
     "vscode-nls": "^5.0.0"
   },
   "dependencies": {
-    "@types/vscode": "1.50.0",
-    "snazzy": "^9.0.0",
-    "ts-standard": "^9.0.0",
+    "@types/vscode": "1.51.0",
+    "snazzy": "file:../node_modules/snazzy",
+    "ts-standard": "file:../node_modules/ts-standard",
```

Comment:
What's the reason for these `file:` dependencies? The Node.js `require()` function recursively searches `node_modules` folders up to the root. I think there may be no need to declare these here. But please let me know if that's not the case.

Reply:
We run `ts-standard` inside `client` and `server` folder, if we execute `ts-standard` at the root, there are no problems but if we want to execute only `ts-standard` inside 1 of the 2 folders, npm throw that error :
```
sh: 1: snazzy: not found
sh: 1: ts-standard: not found
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! client@1.1.8 format: `ts-standard --fix | snazzy`
```
That's why I'm using `file:` dependencies, so we can individually check if `client` or `server` follows the `ts-standard` linting rules, instead of always both.